src/deepsparse/transformers/pipelines/text_classification.py

```diff
@@ -132,20 +139,31 @@ class TextClassificationPipeline(TransformersPipeline):
         If a list of lengths is provided, then for each length, a model and
         tokenizer will be compiled capable of handling that sequence length
         (also known as a bucket). Default is 128
-    :param return_all_scores: if True, instead of returning the prediction as the
+    :param top_k: number of labels with the highest score to return. Default is 1
+    :param return_all_scores: [DEPRECATED] set top_k to desired value instead.
```

Comment:
Under the `return_all_scores` `@property`, I'd recommend adding `warnings.warn('The use of return_all_scores is deprecated. Please use top_k instead ', DeprecationWarning)`

Reply:
we only want to warn for deprecation once - if return all scores is set, then the warning will happen at construction time